docs/auth/usage/index.md

```diff
@@ -110,10 +112,11 @@ allowing you to integrate with other services such as Analytics by providing a u
 Ensure the "Anonymous" sign-in provider is enabled on the [Firebase Console](https://console.firebase.google.com/project/_/authentication/providers).
 
 ```js
-import auth from '@react-native-firebase/auth';
+import { getAuth, signInAnonymously } from '@react-native-firebase/auth';
+
+const auth = getAuth();
 
-auth()
-  .signInAnonymously()
+signInAnonymously()
```

Comment:
needs a `getAuth()` (or `auth` if you made a local variable) in there

In general I don't think the local variable is that useful by the way - I just call `getAuth()` in-place in all the modular API calls, or in a more general sense, for all module's modular calls I call `get<Module>()` right in place, I don't make local variables anywhere

Reply:
> needs a getAuth() (or auth if you made a local variable) in there

Good catch, thanks!

> In general I don't think the local variable is that useful by the way - I just call getAuth() in-place in all the modular API calls, or in a more general sense, for all module's modular calls I call get<Module>() right in place, I don't make local variables anywhere

Good point, personally, I do the same. I only went with the local variables because the docs already made use of a local `authInstance` variable. I just removed all these local vars. :+1: